packages/config/config/devices/0x027a/zen51.json

```diff
@@ -0,0 +1,82 @@
+{
+	"manufacturer": "Zooz",
+	"manufacturerId": "0x027a",
+	"label": "ZEN51",
+	"description": "Z-Wave Plus 700 Series Dry Contact Relay",
+	"devices": [
+		{
+			"productType": "0x0104",
+			"productId": "0x0201"
+		}
+	],
+	"firmwareVersion": {
+		"min": "0.0",
+		"max": "255.255"
+	},
+	"associations": {
+		"1": {
+			"label": "Lifeline",
+			"maxNodes": 5,
+			"isLifeline": true
+		},
+		"2": {
+			"label": "On/Off Control",
+			"maxNodes": 5
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"$import": "~/templates/master_template.json#enable_led_indicator"
+		},
+		{
+			"#": "2",
+			"$import": "templates/zooz_template.json#auto_timer_variable_unit_base",
+			"label": "Auto Off Timer"
+		},
+		{
+			"#": "3",
+			"$import": "templates/zooz_template.json#auto_timer_variable_unit_base",
+			"label": "Auto On Timer"
+		},
+		{
+			"#": "10",
+			"$import": "templates/zooz_template.json#auto_timer_unit_select",
+			"label": "Timer Unit"
```

Comment:
There is already a label in the template you're adding, right? Please just use make the template the one and use that one.